### ISSUE TITLE
Error connecting to TheTVDB when attempting to add a new show

### ISSUE THREAD

**Issue by feelerman** [ltguillaume/droidshows#54]
Subject explains the issue.
Using latest F-Droid release under LineageOS.
No permissions denied from the application.

**Comment by Ibuprophen** [comment]
TheTVDB server has been experiencing some difficulties. They hope to get it back up within the next few days.

**Comment by Ibuprophen** [comment]
TheTVDB Server is up and, appears, to be working now. 

Try it now @feelerman...

**Comment by ltguillaume** [comment]
Seems to be working fine, yeah.

**Comment by cvzi** [comment]
Since a few hours their servers seem to be having issues again 
http://thetvdb.com/api/GetSeries.php?seriesname=breaking+bad gives either a 502 or 503 error

**Comment by ltguillaume** [comment]
Confirmed

**Comment by Ibuprophen** [comment]
@cvzi & @ltGuillaume... 

TheTVDB Guys had just announced yesterday that they are launching a new and updated website. They had later added the following statement... 

"The site will be down briefly while the server is reconfigured. This should increase site availability and reliability."

I had also looked into the API v1 to v2 regarding this as well. I had found that the new website is a preparation for the API v2 but, there's no updated plans for stopping the API v1 at this point in time. 

So bottom line... The very recent launch new site will result in the server going down and up, probably, for the next few days or so but, the API v1 still remains on the server until further notice. 

I hope that I had explained this okay via text... :-) 


~Ibuprophen

**Comment by ltguillaume** [comment]
Then it's a good thing DroidShows is all about having the info at hand while offline :P
